### PR TITLE
JS2 Week1 Assignment-Semih

### PR DESCRIPTION
**Pull request by sigma21**

### CODE REVIEW

File: apps.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+const root = document.querySelector('body');
+const bookList = document.createElement('ul');
+const header = document.createElement('h1');
+header.innerHTML = 'The Best Books of 2018 by Amazon';
+root.appendChild(bookList);
+bookList.appendChild(header);
+
+for (let i = 0; i < books.length; i++) {
+  const listItem = document.createElement('li');
```
Comment (rohni) [inline apps.js]: I would suggest putting all the <li> generating code in its own function, and then just call the function in the loop.

File: apps.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  const cover = document.createElement('img');
+  const title = document.createElement('h2');
+  const author = document.createElement('h3');
+  const language = document.createElement('h4');
+
+  cover.setAttribute('src', bookCovers[books[i]]);
+  title.innerHTML = bookObj[books[i]].title;
+  author.innerHTML = bookObj[books[i]].author;
+  language.innerHTML = bookObj[books[i]].language;
+  listItem.style.border = '2px solid #950740';
```
Comment (rohni) [inline apps.js]: This should be done in a css class.

Comment (sigma21) [reply apps.js]: Noted! Will separate the css.

File: apps.js (hunk 1 above)
Comment (sigma21) [reply apps.js]: Sounds good, I'll try to implement that. Thanks for the idea!

**Comment by sigma21** [comment]
Thanks a lot @rohni for the feedback!

**Pull request closed without merging**